pkg/controller/resources/route.go

```diff
@@ -25,12 +25,22 @@ const (
 
 var (
 	// ErrNotSupportedTLSTermination is an error when unsupported TLS termination is configured via annotation.
-	ErrNotSupportedTLSTermination = errors.New("Not supported tls termination is specified. Only \"passthrough\" is valid.")
+	ErrNotSupportedTLSTermination = errors.New("not supported tls termination is specified. Only \"passthrough\" is valid")
```

Comment:
I suppose this is following Golang rules to not begin an error message with a capital letter? Having the first sentence starting with normal letter and then the second with a capital letter seems strange to me. I guess I just don't undestand the Golang rules.